### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by jnwang** [sfu-db/dataprep#642] [Q&A]
A user tried to install DataPrep, but encountered the following error.

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
anaconda-project 0.9.1 requires ruamel-yaml, which is not installed.
sweetviz 2.1.0 requires jinja2>=2.11.1, but you have jinja2 2.10.3 which is incompatible.

I've shared this post with the user.

**Comment by dovahcrow** [comment]
I would suggest he create a new venv and install dataprep in it. From the error message, I don't see anything related to dataprep.
